### ISSUE TITLE
Skip the landing page if there is only one model.

### ISSUE THREAD

**Issue by lloydtabb** [malloydata/malloy-composer#87]
If there is only one dataset, when I go to this page 

![image](https://user-images.githubusercontent.com/1093458/205324922-c6ac3190-8f89-4dd6-a99a-643132da1720.png)

I should just redirect to

![image](https://user-images.githubusercontent.com/1093458/205324970-b9df8493-9ef6-45b0-9351-9ed28ad87516.png)

**Comment by lloydtabb** [comment]
Id love to get this fixed.   It shows up here and I'm not sure people are finding the 'View'

https://twitter.com/TarasNovak/status/1598689216564760577

**Comment by RandomFractals** [comment]
Good idea. I should have shared this app url instead: https://randomfractals.github.io/chicago-crimes/apps/malloy-composer/#/default?page=about
